backend/sirius-web-diagrams/src/main/java/org/eclipse/sirius/web/diagrams/description/DiagramDescription.java

```diff
@@ -182,6 +197,7 @@ public DiagramDescription build() {
             DiagramDescription diagramDescription = new DiagramDescription();
             diagramDescription.id = Objects.requireNonNull(this.id);
             diagramDescription.label = Objects.requireNonNull(this.label);
+            diagramDescription.autoLayout = Objects.requireNonNull(this.autoLayout);
```

Comment:
Same remark: `requireNonNull` is not needed for primitives.

Reply:
Done